tests/func/experiments/test_show.py

```diff
@@ -471,3 +492,46 @@ def test_show_with_broken_repo(tmp_dir, scm, dvc, exp_stage, caplog):
 
     paths = ["workspace", "baseline", "error"]
     assert isinstance(get_in(result, paths), YAMLFileCorruptedError)
+
+
+def test_show_csv(tmp_dir, scm, dvc, exp_stage, capsys):
```

Comment:
Do we need this test? Can this test be replaced with mock test that checks if `show_experiments` is being called correctly? WDYT? No strong opinion though.

Eg:
https://github.com/iterative/dvc/blob/2b63cedfcd89484bf1ec93e86610e3009b4fa6da/tests/unit/command/test_experiments.py#L61

Reply:
Agreed, In my previous version, what I planned is that we only test `_show_csv` (you asked why I had this function) being called, the value of `all_experiments` (already tested properly), and the `to_csv` function. But I didn't test the code between `all_experiments` and `to_csv` in `show_experiments` in it. And for now as the `show_experiments` had been tested fully, we can just test the call from interface to the `show_experiments`.